app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -278,13 +282,17 @@ class StatePlayerRecyclerViewAssembler private constructor(
       }
     }
 
+    val isTerminalState = ephemeralState.stateTypeCase == StateTypeCase.TERMINAL_STATE
```

Comment:
If we move this higher in the method it might also be able to be used in the conditional above, too.

Reply:
Done